assets/js/components/KeyMetrics/ConversionReportingNotificationCTAWidget.test.js

```diff
@@ -333,5 +336,65 @@ describe( 'ConversionReportingNotificationCTAWidget', () => {
 				KM_ANALYTICS_TOP_TRAFFIC_SOURCE_DRIVING_LEADS,
 			] );
 		} );
+
```

Comment:
Let's expand the test coverage a bit, like we have for tailored metrics variation. Add new group `describe( 'Existing users with manually selected metrics', () => {...}` and move this test case under it, and let's test conditionals for showing the notification like:
* `does not render when there are no metrics selected`: user input completed - with tailored metrics, no manual metrics are saved, purpose differs from newly detected events (to prevent tailored metrics notification), like `sell_products` as a purpose and `contact` as new detected events (already globally provided by default via `receiveConversionReportingInlineData`)
* `does not render when new events bring the metrics user already selected`: include few leads metrics as saved ones (`provideKeyMetrics` util), like `KM_ANALYTICS_TOP_CITIES_DRIVING_LEADS`, `KM_ANALYTICS_TOP_PAGES_DRIVING_LEADS` and `KM_ANALYTICS_TOP_TRAFFIC_SOURCE_DRIVING_LEADS`
* `does not render when isKeyMetricsSetupCompleted is not true`:
* `renders when there are new events with metrics user have not selected`: here we can verify that when leads metrics are provided include few leads metrics as saved ones (`provideKeyMetrics` util), like `KM_ANALYTICS_TOP_CITIES_DRIVING_LEADS`, `KM_ANALYTICS_TOP_PAGES_DRIVING_LEADS` but not `KM_ANALYTICS_TOP_TRAFFIC_SOURCE_DRIVING_LEADS` for example, banner should still show as there is one metrics user does not have as selected, and also as a second case, having all the leads metrics and including `registry.dispatch( MODULES_ANALYTICS_4 ).setDetectedEvents( [ 'contact', 'add_to_cart' ] );` which should show the banner, etc
* `renders if user input have been completed and then user switched to the manual selection`: Set user input completed to `true` and provide manual key metrics not including ACR metrics, banner should still surface with new events